aspnetcore/blazor/hosting-models.md

```diff
@@ -492,6 +492,26 @@ To configure the SignalR client in the *Pages/_Host.cshtml* file:
 </script>
 ```
 
+## Conditionally execute code based on location
```

Comment:
@SteveSandersonMS @lewing is there any additional guidance that we want to provide here about when it's appropriate to do this?

Reply:
@guardrex I think we should hold off on documenting this technique for platform detection. I'm not convinced the runtime is doing what we want here: https://github.com/mono/mono/issues/18627.